lib/aquatone/collectors/crtsh.rb

```diff
@@ -0,0 +1,21 @@
+module Aquatone
+  module Collectors
+    class Crtsh < Aquatone::Collector
+      self.meta = {
+        :name         => "crtsh",
+        :author       => "Joel (@jolle)",
+        :description  => "Uses crt.sh by COMODO CA to find hostnames"
+      }
+
+      def run
+        response = get_request("https://crt.sh/?dNSName=%25.#{url_escape(domain.name)}")
+
+        parsed = parse_html(response.body)
+
+        parsed.css('table').css('table').css('tr').css('td:nth-child(4)').map do |column|
```

Comment:
Sure. I decided to use Nokogiri because [it is not generally recommended](https://stackoverflow.com/a/1732454) to use regex when parsing HTML, but I'll remove Nokogiri as a dependency and use regex.